src/main/kotlin/com/ekino/oss/errors/handler/CoreExceptionHandler.kt

```diff
@@ -8,6 +8,7 @@ import com.ekino.oss.errors.generator.notFound
 import com.ekino.oss.errors.generator.unavailable
 import com.ekino.oss.errors.generator.unsupportedMediaType
 import com.ekino.oss.errors.property.ErrorsProperties
+import com.fasterxml.jackson.databind.exc.InvalidFormatException
```

Comment:
unfortunately it adds a strong dependency with Jackson. since Jackson is now a norm with Spring MVC I don't think it is such a big problem

Reply:
I agree.